[Bug] Does not compile on ubuntu 26.04

### Description

Compilation fails in a chroot.
Todays build of git master fails in a chroot (sbuild) container for the package

```
BuddiesOfBudgie/budgie-desktop

main

GITHUB

I am getting this compilation error

FAILED: [code=1] src/panel/budgie-panel.p/meson-generated_settings_settings_style.c.o 
cc -Isrc/panel/budgie-panel.p -Isrc/panel -I../src/panel -Isrc/plugin/panel -I../src/plugin/panel -Isrc/config -I../src/config -Isrc/libsession -I../src/libsession -Isrc/lib -I../src/lib -Isrc/panel/plugin -I../src/panel/plugin -Isrc/theme -I../src/theme -Isrc/plugin/raven -I../src/plugin/raven -Isrc/raven -I../src/raven -Isrc/windowing -I../src/windowing -Isubprojects/gvc -I../subprojects/gvc -I/usr/include/gio-unix-2.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/glycin-2 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -I/usr/include/gtk-layer-shell -I/usr/include/libpeas-2 -I/usr/include/uuid -I/usr/include/xfce4 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=c11 -w -Wno-incompatible-pointer-types -Wno-implicit-function-declaration '-DGETTEXT_PACKAGE="budgie-desktop"' -g -O2 -Werror=implicit-function-declaration -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/build/reproducible-path/budgie-desktop-10.10.1+20260225=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection '-fdebug-prefix-map=/build/reproducible-path/budgie-desktop-10.10.1+20260225=/usr/src/budgie-desktop-10.10.1+20260225-0ubuntu0~ppa' -Wdate-time -D_FORTIFY_SOURCE=3 -pthread -D_REENTRANT -MD -MQ src/panel/budgie-panel.p/meson-generated_settings_settings_style.c.o -MF src/panel/budgie-panel.p/meson-generated_settings_settings_style.c.o.d -o src/panel/budgie-panel.p/meson-generated_settings_settings_style.c.o -c src/panel/budgie-panel.p/settings/settings_style.c
src/panel/budgie-panel.p/settings/settings_style.c: In function ‘budgie_style_page_set_labwc_theme’:
src/panel/budgie-panel.p/settings/settings_style.c:1216:25: error: implicit declaration of function ‘symlink’ [-Wimplicit-function-declaration]
 1216 |                         symlink (_tmp19_, _tmp20_);
      |                         ^~~~~~~
```

This is choking on settings_style.vala for the FileUtils.symlink method.

I can workaround this in two ways:

1. patch FileUtils.symlink to use File.make-symbolic-link
2. patch root meson.build to not specify the compiler standards version c11  with this patch

```

--- a/meson.build
+++ b/meson.build
@@ -6,10 +6,6 @@
         'GPL-2.0',
         'LGPL-2.1',
     ],
-    default_options: [
-        'c_std=c11',
-        'warning_level=3'
-    ],
 )
 
 # Vala generates bad C code and missing these on gcc 14 will cause FTBFS
@@ -18,7 +14,7 @@
 am_cflags = [
     '-w',
     '-Wno-incompatible-pointer-types',
-    '-Wno-implicit-function-declaration',
+#    '-Wno-implicit-function-declaration',
 ]
 add_global_arguments(am_cflags, language: 'c')
 
```

C11 is relatively old- I think ubuntu is now on C17



### Budgie version

10.10.1 + git master

### Operating System

UB 26.04

### Steps to reproduce the issue

build budgie desktop for 26.04 via sbuild

### Actual result

see the description

### Expected result

clean build

### Additional information

_No response_

--- a/meson.build
+++ b/meson.build
@@ -6,10 +6,6 @@ project(
         'GPL-2.0',
         'LGPL-2.1',
     ],
-    default_options: [
-        'c_std=c11',
-        'warning_level=3'
-    ],
 )
 
 # Vala generates bad C code and missing these on gcc 14 will cause FTBFS

